helix-core/src/syntax.rs

```diff
@@ -716,8 +716,11 @@ impl LanguageLayer {
 // For now cheat and just throw out non-root layers if they exist. This should still improve
 // parsing in majority of cases.
 
-use std::sync::atomic::{AtomicUsize, Ordering};
-use std::{iter, mem, ops, str, usize};
+use std::{
+    iter, mem, ops, str,
+    sync::atomic::{AtomicUsize, Ordering},
+    usize,
+};
```

Comment:
Why not separate them?

Reply:
I briefly mentioned this in the channel: This is just the default `rustfmt` settings clashing with the ones that were used for the project thus far. I think it's a good idea if the formatting options for this project are commited in, so there's no noise in the diffs.